doc/developer-notes.md

```diff
@@ -575,6 +575,54 @@ llvm-cov show \
 
 The generated coverage report can be accessed at `build/coverage_report/index.html`.
 
+#### Compiling for Fuzz Coverage
+
+```shell
+cmake -B build \
```

Comment:
how do you feel about naming the dir something more usage specific like `build_fuzz_cov` since devs will likely be using build for other things?

Reply:
I have been following the docs since `build` was used before with lcov coverage . https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#using-lcov

But if others agree with this , I will change it and might have to retouch the llvm/clang based coverage report generation too then .